recipes/qibo/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "qibo" %}
+{% set version = "0.0.1b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Quantum-TII/qibo/archive/v{{ version }}.tar.gz
+  sha256: 245524833067afe98a83c9ca1f6e8559cc4feba8f2271d130b7e230ebac77371
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  missing_dso_whitelist:
+    - $RPATH/libtensorflow_framework.so.2
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - pip
+    - tensorflow >2.1
+  run:
+    - python
+    - joblib
+    - matplotlib
```

Comment:
this really does need to be `matplotlib-base`

Reply:
Thanks.